packages/default/scss/expansion-panel/_layout.scss

```
@@ -58,9 +58,22 @@
         box-sizing: border-box;
     }
 
-    .k-expander.k-expanded + .k-expander,
-    .k-expander.k-expanded:not(:first-child) {
-        margin-top: $expander-spacing-y;
+    .k-expander.k-expanded,
+    .k-expander + .k-expander.k-expanded {
+        margin-bottom: $expander-spacing-y;;
+    }
+
+    .k-expander + .k-expanded {
+        margin-top: $expander-spacing-y;;
+    }
+
+    .k-expander.k-expanded + .k-expanded {
+        margin-top: 0;
+    }
+
+    .k-expander.k-expanded:last-child,
+    .k-expander + .k-expander.k-expanded:last-child{
+        margin-bottom: 0;
```

You can safely change the code to:

```suggestion
    .k-expander.k-expanded {
        margin-top: $expander-spacing-y;
        margin-bottom: $expander-spacing-y;
    }
    .k-expander.k-expanded:first-child {
        margin-top: 0;
    }
    .k-expander.k-expanded:last-child {
        margin-bottom: 0;
```

Top and bottom margin collapse in normal flow, however we have `display: grid` in the test page and that's why they don't collapse. I will update the test

---

@joneff, Changing the selectors to the suggested ones will not handle the initial issue illustrated her https://www.telerik.com/kendo-angular-ui/components/layout/expansionpanel/icons/

---

I've updated my suggestion